etc/openstack-config/openstack-config.yml

```diff
@@ -399,116 +369,93 @@ hpc_v1_32cpu:
+    resources:CUSTOM_COMPUTE_HIGHMEM: 1
 
-hpc_v1_64cpu:
-  name: "hpc.v1.64cpu"
-  ram: 262144
+# HPC v2:
+# Core-pinned VCPUs
+# 8GB RAM per VCPU
+# Spread across NUMA regions
+# General purpose small
+hpc_v2_8cpu_64ram:
+  name: "hpc.v2.8cpu.64ram"
+  ram: 65536
   disk: 30
-  vcpus: 64
-  is_public: false
+  vcpus: 8
+  is_public: true
   extra_specs:
     hw:cpu_policy: "dedicated"
-    hw:cpu_thread_policy: "prefer"
-    hw:cpu_threads: 2
     hw:mem_page_size: "1GB"
     hw:cpu_sockets: 2
     hw:numa_nodes: 8
     hw:pci_numa_affinity_policy: preferred
     hw_rng:allowed: "True"
+    trait:CUSTOM_COMPUTE_AUXILIARY: "required"
+    resources:CUSTOM_COMPUTE_AUXILIARY: 1
 
-hpc_v1_80cpu:
-  name: "hpc.v1.80cpu"
-  ram: 327680
+# HPC v2:
+# Core-pinned VCPUs
+# 8GB RAM per VCPU
+# Spread across NUMA regions
+# General purpose large
+hpc_v2_16cpu_128ram:
+  name: "hpc.v2.16cpu.128ram"
+  ram: 131072
   disk: 30
-  vcpus: 80
-  is_public: false
+  vcpus: 16
+  is_public: true
   extra_specs:
     hw:cpu_policy: "dedicated"
-    hw:cpu_thread_policy: "prefer"
-    hw:cpu_threads: 2
     hw:mem_page_size: "1GB"
     hw:cpu_sockets: 2
     hw:numa_nodes: 8
     hw:pci_numa_affinity_policy: preferred
     hw_rng:allowed: "True"
+    trait:CUSTOM_COMPUTE_AUXILIARY: "required"
+    resources:CUSTOM_COMPUTE_AUXILIARY: 2
 
-hpc_v1_96cpu:
-  name: "hpc.v1.96cpu"
-  ram: 393216
+# HPC v2:
+# Core-pinned VCPUs
+# 8GB RAM per VCPU
+# Spread across NUMA regions
+# GPU A100
+hpc_v2_16cpu_128ram_a100:
+  name: "hpc.v2.16cpu.182ram.a100"
+  ram: 131072
   disk: 30
-  vcpus: 96
+  vcpus: 16
   is_public: false
   extra_specs:
     hw:cpu_policy: "dedicated"
-    hw:cpu_thread_policy: "prefer"
-    hw:cpu_threads: 2
     hw:mem_page_size: "1GB"
     hw:cpu_sockets: 2
     hw:numa_nodes: 8
     hw:pci_numa_affinity_policy: preferred
     hw_rng:allowed: "True"
-
-vgpu_a100d_2g_20gb:
-  name: "vgpu.a100d.2g.20gb"
-  ram: 32768
-  disk: 30
-  vcpus: 8
-  is_public: true
-  extra_specs:
-    hw:cpu_policy: "dedicated"
-    hw:cpu_thread_policy: "prefer"
-    hw:mem_page_size: "1GB"
-    hw:cpu_sockets: 2
-    hw:numa_nodes: 4
-    hw_rng:allowed: "True"
-    resources:CUSTOM_NVIDIA_700: "1"
-
-vgpu_a100d_1g_10gb:
-  name: "vgpu.a100d.1g.10gb"
-  ram: 16384
-  disk: 30
-  vcpus: 4
-  is_public: true
-  extra_specs:
-    hw:cpu_policy: "dedicated"
-    hw:cpu_thread_policy: "prefer"
-    hw:mem_page_size: "1GB"
-    hw:cpu_sockets: 2
-    hw:numa_nodes: 2
-    hw_rng:allowed: "True"
-    resources:CUSTOM_NVIDIA_699: "1"
-
-vgpu_a100d_40c:
-  name: "vgpu.a100d.40c"
-  ram: 65536
-  disk: 30
-  vcpus: 16
-  is_public: true
-  extra_specs:
-    hw:cpu_policy: "dedicated"
-    hw:cpu_thread_policy: "prefer"
-    hw:mem_page_size: "1GB"
-    hw:cpu_sockets: 2
-    hw:numa_nodes: 8
-    hw_rng:allowed: "True"
-    resources:CUSTOM_NVIDIA_697: "1"
-
+    pci_passthrough:alias: "a100:2"
+    trait:CUSTOM_COMPUTE_GPU_A100: "required"
```

Comment:
Didn't we say one card per VM? That would match your resource.

Reply:
Yeah that's right, thanks. Done.